CHANGELOG.md

```diff
@@ -1,23 +1,64 @@
-x.y.z Release notes (yyyy-MM-dd)
+10.0.0-beta.3 Release notes (yyyy-MM-dd)
+
 =============================================================
+
 ### Enhancements
-* None.
+
+* The following classes & aliases have been renamed:
+
+`RLMSyncUser` -> `RLMUser`
+`SyncUser` -> `User`
+
+`RLMAppCredential` -> `RLMCredential`
+`AppCredential` -> `Credential`
+
+`RealmApp` -> `App`
+
+`RLMUserAPIKeyProviderClient` -> `RLMAPIKeyAuth`
+`UserAPIKeyProviderClient` -> `APIKeyAuth`
+
+`RLMUsernamePasswordProviderClient` -> `RLMEmailPasswordAuth`
+`UsernamePasswordProviderClient` -> `EmailPasswordAuth`
+
+* The following functionality has also moved to the User
+
+`[RLMApp callFunctionNamed:]` -> `[RLMUser callFunctionNamed:]`
+`App.functions` -> `User.functions`
+
+`[RLMApp mongoClientWithServiceName:]` - > `[RLMUser mongoClientWithServiceName:]`
+`App.mongoClient(serviceName)` -> `User.mongoClient(serviceName)`
+
+`[RLMApp userAPIKeyProviderClient]` -> `[RLMUser apiKeyAuth]`
+`App.userAPIKeyProviderClient` -> `App.apiKeyAuth()`
+
+`[RLMApp logOut:]` -> `[RLMUser logOut]`
+`App.logOut(user)` -> `User.logOut()`
+
+`[RLMApp removeUser:]` -> `[RLMUser remove]`
+`App.remove(user)` -> `User.remove()`
+
+`[RLMApp linkUser:credentials:]` -> `[RLMUser linkWithCredentials:]`
+`App.linkUser(user, credentials)` -> `User.link(credentials)`
+  
+* Other enhancements
+
+-  `refreshCustomData()` on User now returns the custom data on success.
+
+The purpose of these changes is to align Cocoa with the other Realm SDK's
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-cocoa/issues/????), since v?.?.?)
 * None.
 
-<!-- ### Breaking Changes - ONLY INCLUDE FOR NEW MAJOR version -->
-
 ### Compatibility
+* This release introduces breaking changes w.r.t some sync classes and MongoDB Realm Cloud functionality. 
+(See enchancements for full list)
 * File format: Generates Realms with format v11 (Reads and upgrades all previous formats)
 * Realm Studio: 10.0.0 or later.
 * APIs are backwards compatible with all previous releases in the 5.x.y series.
 * Carthage release for Swift is built with Xcode 11.5.
 
 ### Internal
-* Upgraded realm-core from ? to ?
-* Upgraded realm-sync from ? to ?
+* None
```

Comment:
Sections with no relevant changes are removed as part of the release process, not in unrelated PRs.